Tasks/PublishSymbols/Helpers.ps1

```diff
@@ -1,40 +1,3 @@
-[cmdletbinding()]
```

Comment:
I renamed Invoke-IndexSources.ps1 to Helpers.ps1 and reorganized it into helper functions for testability.